src/Medidata.ZipkinTracer.Core.Collector/Properties/AssemblyInfo.cs

```diff
@@ -32,7 +32,7 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("0.0.6")]
-[assembly: AssemblyFileVersion("0.0.6")]
-[assembly: AssemblyInformationalVersion("0.0.6")]
+[assembly: AssemblyVersion("0.0.7")]
```

Comment:
I am not sure if this is the version of the nuget but given that it stopped doing what used to do, this is a new major version.